### PR TITLE
Grammatical Tweaks (Section 07)

### PR DESCRIPTION
**Pull request by furey**
Following #328, this PR applies numerous grammatical tweaks to section 07 of the documentation.

### CODE REVIEW

File: 07-Database/02-Query-Builder.adoc [hunk 1]
```diff
@@ -187,19 +199,19 @@ await Database.from('users').whereNotExists(function () {
 ----
 await Database
   .table('users')
-  .whereBetween('age',[18,32])
+  .whereBetween('age', [18,32])
```
Comment (RomainLanz) [inline 07-Database/02-Query-Builder.adoc]:
```suggestion
  .whereBetween('age', [18, 32])
```

File: 07-Database/02-Query-Builder.adoc [hunk 2]
```diff
@@ -187,19 +199,19 @@ await Database.from('users').whereNotExists(function () {
...
 ----
 
 ==== whereNotBetween
 [source, javascript]
 ----
 await Database
   .table('users')
-  .whereNotBetween('age',[45,60])
+  .whereNotBetween('age', [45,60])
```
Comment (RomainLanz) [inline 07-Database/02-Query-Builder.adoc]:
```suggestion
  .whereNotBetween('age', [45, 60])
```

File: 07-Database/03-Migrations.adoc [hunk 3]
```diff
@@ -60,46 +58,122 @@ class UsersSchema extends Schema {
...
-The `up` method is used to take action on a table. It can be creating a new table or altering the existing table.
+The `up` method is used to take action on a table. It can be used to create a new table or altering an existing table.
 
-down::
-The `down` method is the reverse of the up action. When `up` method creates a table, you drop it inside the `down` method.
+=== down()
 
-== Multiple connections
-The schema files can also make use of a different connection by defining a `connection` property on them. Also, make sure that the same connection is defined inside `config/database.js` file.
+The `down` method is used to revert the changes applied in the `up` method. When `up` is used to create a table, `down` would be used to  drop that table.
```
Comment (RomainLanz) [inline 07-Database/03-Migrations.adoc]:
```suggestion
The `down` method is used to revert the changes applied in the `up` method. When `up` is used to create a table, `down` would be used to drop that table.
```

**Review by RomainLanz — CHANGES_REQUESTED** [review]
WOW! 🎉 

Thanks again for your contribution. 💜 
I've only spotted 3 changes, everything else is perfect.

**Comment by furey** [comment]
@RomainLanz No worries, cheers! 👍

**Pull request merged**